keyboard/src/data/keyboard_raw.ycp

```diff
@@ -230,13 +230,13 @@ return ($[
 	$[
 	    "pc104"	: $[ "ncurses": "us-intl.map.gz"],
 	    "macintosh" : $[ "ncurses": "us-mac.map.gz"],
-	    "sparc64"	: $[ "type4": $[ "ncurses": "us.map.gz"],
+	    "type4"	: $[ "ncurses": "us.map.gz"],
 	    "type5"	: $[ "ncurses": "us.map.gz" ],
 	    "type5_euro": $[ "ncurses": "us.map.gz" ],
 	]
     ],
-  ],
```

Comment:
Oh, at first sight it looked like a mistake to remove this, but it was actually part of the unclosed `sparc64` key, which was a bug I guess...